virus/partitivirus_amanitinum.md

```diff
@@ -0,0 +1,148 @@
+# {Q1A Paper Title}
+written by: [Gretl Baghdadi](https://github.com/gretlb04)
+
+[YouTube/Video Link to your presentation video] https://youtu.be/qWHaekskyBo 
+
+### {Q1B Virus Name}
```

Comment:
Remove the Generic headers of the Repo report file and make them specific to your work. Anything in squiggle brackets `{` should not be included.